CMakeLists.txt

```
@@ -34,6 +35,9 @@ cmake_dependent_option(DESUL_ENABLE_SYCL "Build with SYCL support" ON "ENABLE_SY
 cmake_dependent_option(DESUL_ENABLE_TESTS "Build tests" ON "ENABLE_TESTS" OFF)
 
 include(cmake/blt_boilerplate.cmake)
+if (CMAKE_CXX_STRANDARD LESS 17)
```

```suggestion
if (CMAKE_CXX_STANDARD LESS 17)
```

God I wish cmake had linting for things like this.